### PR TITLE
[RHACS] Added a note auto-generated registries are disabled in ACS

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-25058

Added a note.

Preview: https://79031--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-image-registries.html#automatic-configuration-image-registry_integrate-with-image-registries

Cherry-pick into:
- `rhacs-docs-4.4`
- `rhacs-docs-4.5`

### CODE REVIEW

File: modules/automatic-configuration-image-registry.adoc [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
+= Automatic configuration
 
 [role="_abstract"]
 {product-title} includes default integrations with standard registries, such as Docker Hub and others. It can also automatically configure integrations based on artifacts found in the monitored clusters, such as image pull secrets. Usually, you do not need to configure registry integrations manually.
 
 [IMPORTANT]
 ====
-If you are using a GCR registry, {product-title} does not create a registry integration automatically.
+* If you are using a GCR registry, {product-title} does not create a registry integration automatically.
+* If you are using {product-title-managed}, automatic configuration is not available. You must create registry integrations manually.
```
Comment (shipsing) [inline modules/automatic-configuration-image-registry.adoc:14]:
```suggestion
* If you use a Google Container Registry (GCR), {product-title} does not create a registry integration automatically.
* If you use {product-title-managed}, you must create a registry integration manually.
```

**Review by shipsing — CHANGES_REQUESTED** [review]
@gaurav-nelson, Provided a suggestion for minimalism. Thanks

File: modules/automatic-configuration-image-registry.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/automatic-configuration-image-registry.adoc]: Thank you @shipsing for checking this one. I've updated the first bullet point. However, for the second one, I want to highlight that the feature `Automatic configuration` is not available in `product-title-managed`. Therefore, I've stated the name. I've updated the sentence to the following:
```
If you use {product-title-managed}, automatic configuration is unavailable, 
and you must manually create registry integrations.
```

**Pull request merged**